src/utils/app-init.ts

```diff
@@ -0,0 +1,27 @@
+import { KeycloakService } from 'keycloak-angular';
```

Comment:
Why is this

Reply:
we are tring for handling keycloak for private module right in that case i used